ulib/experimental/FStar.Reflection.Typing.fsti

```diff
@@ -1778,18 +1778,20 @@ type sigelt_typing : env -> sigelt -> Type0 =
   | ST_Let :
     g : env ->
     fv : R.fv ->
+    us: list R.univ_name ->
     ty : R.typ ->
     tm : R.term ->
     squash (typing g tm (E_Total, ty)) ->
-    sigelt_typing g (pack_sigelt (Sg_Let false [pack_lb ({ lb_fv = fv; lb_us = []; lb_typ = ty; lb_def = tm })]))
+    sigelt_typing g (pack_sigelt (Sg_Let false [pack_lb ({ lb_fv = fv; lb_us = us; lb_typ = ty; lb_def = tm })]))
```

Comment:
What happens if `ty` has unbound universe names? Will it be silently accepted? There's other places where universes are bogus (e.g. lookup_fvar_uinst), it's probably quite a bit of work to make the entire judgment correct wrt to universes, but at least a note would help.

Reply:
I'll add a note.

(But I'm not yet convinced that you can actually exploit this: the typing judgment is with regard to the initial environment `g`, which has no bound universes.  Unless you do something terribly hacky (like I did in Pulse), this API still doesn't allow you to add universe-polymorphic splices.  It just doesn't explode on you when you poke under the hood.)